### PR TITLE
[SDK-166] Remove the draft when the back button is clicked.

### PR DESCRIPTION
**Pull request by gadamyan**
## Description

-   Added functionality to call the draft deletion API when the back button is pressed.

#### Added

-   Added functionality to call the draft deletion API when the back button is pressed.

## How to Test

-   Edit existing avatar and without saving, click the back button. Edit the same avatar again, there should not be leftovers from the previous edition.

## Checklist

-   [ ] Changelog is updated.

### CODE REVIEW

File: Source/RpmAvatarCreator/Public/RpmAvatarRequestHandler.h [hunk 1]
```diff
@@ -54,7 +54,7 @@ class RPMAVATARCREATOR_API URpmAvatarRequestHandler : public UObject
 	void OnSaveAvatarCompleted(bool bSuccess, FAvatarSaveCompleted AvatarSaveCompleted, FAvatarCreatorFailed Failed);
 
 	UFUNCTION()
-	void OnDeleteAvatarCompleted(bool bSuccess, FAvatarDeleteCompleted AvatarDeleteCompleted, FAvatarCreatorFailed Failed, FString AvatarId);
+	void OnDeleteAvatarCompleted(bool bSuccess, FAvatarDeleteCompleted AvatarDeleteCompleted, FAvatarCreatorFailed Failed, FString AvatarId, bool bIsDraft);
```
Comment (HarrisonHough) [inline Source/RpmAvatarCreator/Public/RpmAvatarRequestHandler.h:57]: minor thing but I think bSuccess could be changed to bIsSuccessful to match the naming of the bIsDraft. In general I'm a fan of the "isSomething" naming convention for booleans

**Review by HarrisonHough — APPROVED** [review]
Looks good, small suggestion in comments

**Review by HarrisonHough — APPROVED** [review]
Looking good !

File: Source/RpmAvatarCreator/Public/RpmAvatarRequestHandler.h (hunk 1 above)
Comment (gadamyan) [reply Source/RpmAvatarCreator/Public/RpmAvatarRequestHandler.h:57]: bSuccess comes from IHttpRequest unreal public api. It's a forwarded parameter. If I change it I'd need to  change the whole implementation.

**Pull request merged**